Tutorials/Notebooks/netket-VMC.ipynb

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
I would maybe move all existing "tutorials" in a folder Examples/, and keep only notebooks with actual tutorials in Tutotorials/ (without the extra Notebooks/ folder) and with a consistent naming (for example this file should maybe called J1J2 or something like that?)

Reply:
Done